### ISSUE TITLE
audio voices on Raspberry Pi

### ISSUE THREAD

**Issue by chundermike** [liballeg/allegro5#1248]
Hi,

Does allegro support more than one voice in the audio add-on?  I'm trying to output different audio streams to different devices simultaneously (i.e. onboard 2.5mm jack, and DAC/Amp HAT add-on board).

Thanks in advance,
Mike

**Comment by beoran** [comment]
Yes, you can create multiple voices with Puleseaudio or Alsa dirvers. however, I'm not quite sure how to link those to your desired outputs.

**Comment by SiegeLord** [comment]
There's this pull request which lists audio devices: https://github.com/liballeg/allegro5/pull/1238 I don't know if it works w/ RPI, but if it does, we could add an option to select which audio device to use.
